code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -258,21 +258,28 @@
 		var/mob/living/L = M
 		L.add_trait(TRAIT_STUNIMMUNE, id)
 		L.add_trait(TRAIT_SLEEPIMMUNE, id)
+		if(iscarbon(L))
+			var/mob/living/carbon/C = L
+			C.gain_trauma(/datum/brain_trauma/special/psychotic_brawling, TRAUMA_RESILIENCE_LOBOTOMY)
 
 /datum/reagent/drug/bath_salts/on_mob_delete(mob/M)
 	if(isliving(M))
 		var/mob/living/L = M
 		L.remove_trait(TRAIT_STUNIMMUNE, id)
 		L.remove_trait(TRAIT_SLEEPIMMUNE, id)
+		if(iscarbon(L))
+			var/mob/living/carbon/C = L
+			for(var/datum/brain_trauma/special/psychotic_brawling/T in C.get_traumas())
```

Comment:
Might want to make sure there's no edge case where the person already had psychotic brawling on the lobotomy level beforehand, though I'm not entirely certain how you'd go about that. Maybe a variable on the reagent?